now.json

```diff
@@ -0,0 +1,56 @@
+{
+  "version": 2,
+  "builds": [{ "src": "index.js", "use": "@now/node@latest" }],
+  "routes": [
+    {
+      "src": "/stops/nearby",
+      "dest": "/index.js",
+      "methods": ["GET"]
```

Comment:
Is it necessary to explicitly allow `OPTIONS` here for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)?

Reply:
I am not sure what you mean, I can remove `methods` to allow all of the methods (including `OPTIONS`. Is that what you want?

Reply:
Is `OPTIONS` implicitly/automatically allowed with now.sh? Or do we need to add it here explicitly? You can check with `curl -X OPTIONS …`.